libraries/src/test/java/com/baeldung/circularfifoqueue/example/CircularFifoQueueTest.java

```diff
@@ -0,0 +1,161 @@
+package com.baeldung.circularfifoqueue.example;
+
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.commons.collections4.queue.CircularFifoQueue;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class CircularFifoQueueTest {
+
+	private static final int DEFAULT_SIZE = 32;
```

Comment:
Indent the file with spaces, not tabs. There's a standard formatter in the eclipse module of the tutorials repo that you can import if you're using Eclipse.